backend/open_webui/utils/auth.py

```
@@ -8,6 +8,12 @@
 import os
 
 
+from cryptography.hazmat.primitives.ciphers.aead import AESGCM
+from cryptography.hazmat.primitives.asymmetric import ed25519
```

Ruff (0.12.2)

`cryptography.hazmat.primitives.asymmetric.ed25519` imported but unused

Remove unused import: `cryptography.hazmat.primitives.asymmetric.ed25519`

(F401)